harvester/jax.py

```diff
@@ -105,7 +109,18 @@ def convert(jax_evidence):
             }
         }]
         # add summary fields for Display
-        association['evidence_label'] = evidence['response_type']
+        for item in match.ev_lab:
```

Comment:
@mayfielg Thanks for the PR.  Looks pretty great.
Two requests:
* can you open it against v0.3?
* would it be a good idea to collapse the repeated block of code here into a method in match?  The result would be a single line calling the new method in each of the harvesters.